doc/TBG_macros.cfg

```diff
@@ -188,6 +188,30 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# Live in situ visualization using ISAAC
+# --isaac.period PERIOD
+#   ↳ Initial period in which a image shall be rendered
```

Comment:
just plain spaces or something that is already used in the paragraphs above or below